### ISSUE TITLE
Feature request for Select component

### ISSUE THREAD

**Issue by thdoan** [vuetifyjs/vuetify#337]
While the Select component looks nice, it can be improved to offer better keyboard navigation. Things that can be improved (all of these are already possible on a standard `<select>` element):

- You should be able to navigate to the select component using `Tab` and `Shift`+`Tab`.
- If the select component has focus, you should be able to open the menu by pressing `Enter` or `Space`.
- If the select component is open:

  - you should be able to select an item by pressing `Enter`
  - you should be able to close it by pressing `Esc`
  - pressing `Up` should navigate to the previous item, or the last item if on the first item
  - pressing `Down` should navigate to the next item, or the first item if on the last item
  - pressing `PgUp` should navigate to the previous page, or first item if less than a page away
  - pressing `PgDn` should navigate to the next page, or last item if less than a page away
  - pressing `Home` should navigate to the first item
  - pressing `End` should navigate to the last item
  - pressing a letter should go to the first item starting with that letter
  - pressing a letter repeatedly should cycle through each item starting with that letter<br><br>

- The select component should have full ARIA attribution so that people using screen readers can more easily navigate it. You can use Google's own select component from Google Forms as a reference (see Target Language):

  https://docs.google.com/forms/d/e/1FAIpQLSftGMTKlnLwoyTpxE9tHUFjxlFagbqGMZD7qmIB9wNGTrbB7g/viewform

**Comment by johnleider** [comment]
Thank you for your feedback.

*1 You can do
*2 You can do
*3 In progress

This is also a duplicate of #224 for the most part.

**Comment by stoked74** [comment]
Hey John loving vuetify and the recent updates! 

I want to +1 this. The app I'm building is nothing but very large forms and I already know my users are going to be complaining about not being able to tab into the selects and use the arrow keys to navigate up and down once they're opened. You noted above that you should be able to tab to a select but I can't a select to receive focus with the tab key on any of my forms.

A lot of my selects also have a lot of options so the ability to navigate to the first option of the selected letter is also very important.

I appreciate all of your hard work.

**Comment by johnleider** [comment]
I've added most of what was requested here, will release with next release.

**Comment by MitchelLabonte** [comment]
I was happily surprised to see that this issue was closed and tried it in the doc, but it looks like those two points are not done:
```
- pressing a letter should go to the first item starting with that letter
- pressing a letter repeatedly should cycle through each item starting with that letter
```

Is there any plan to implement those as well in a future release?

**Comment by johnleider** [comment]
I think it could only be supported if autocomplete (search) was disabled as they would interfere with each other.

**Comment by MitchelLabonte** [comment]
There is support for autocomplete in selects? I don't see it in the doc, how can it be used?

**Comment by johnleider** [comment]
That's not officially supported yet either, but you can use it with the autocomplete prop.

**Comment by MitchelLabonte** [comment]
Oh I see, thank you

**Comment by monkeybrother65** [comment]
I love the enhancements.  This makes the select component much more usable.  There are a few more things that would make it more polished.

1- when you tab to the select field it does not highlight.  It is confusing to the user.
2- when the select component has the highlight.  The user should be able to hit the delete key and it should null out the selection.  For completeness the delete key should work wether the select drop-down is open or closed.

Thank you!!!

**Comment by johnleider** [comment]
Can you please open a new issue @monkeybrother65

**Comment by thdoan** [comment]
FYI the `Delete` key does nothing in both the [native select](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select) and Google's [Material select](https://docs.google.com/forms/d/e/1FAIpQLSfBeozaGcMlNlh2Zx0_We6weVrp9ivui_enI1vUOo4aOUq5lQ/viewform), so this would be something new.

**Comment by ksenechkaP** [comment]
Still have problem with keyboard navigation. There is no aria attribute, and screen reader "not see" select options. What need I fix? Or this is not implemented in vuetify?  
Thanks in advance!
